src/utils/foundry.ts

```diff
@@ -0,0 +1,10 @@
+import { spawnChild } from '@utils/childprocess'
+
+export const checkIsFoundryInstalled = async () => {
+  try {
+    await spawnChild('which', ['anvil', 'forge'])
+    return true
+  } catch (_error) {
+    throw new Error('Foundry is not installed')
+  }
+}
```

Comment:
Is it possible to encounter other errors that are not related to a foundry/anvil not found error?